### DISCUSSION TITLE
Can a windows machine be used as a snapserver?

### DISCUSSION THREAD

**Discussion by Marshalldog** [snapcast/snapcast#1032] [General]
Can a windows machine be used as a snapserver?
and what setup is required (assuming it's not within the setup guide within the setup documentation)?

**Comment by Marshalldog** [comment]
eg. play say, music on a windows pc, which then gets sent to pi zero (snapclients)????

**Comment by Slaughthammer** [comment]
Any news on this? Searching for a solution as well.

**Comment by sparky073** [comment]
Been trying this for a couple days but I'm not knowledgeable enough to troubleshoot the problems I'm having. Following this guide
https://github.com/badaix/snapcast/blob/develop/doc/build.md#windows-vcpkg

**Comment by badaix** [comment]
Snapserver does not run nativly on Windows, see also [CMakeLists.txt](https://github.com/badaix/snapcast/blob/develop/CMakeLists.txt): 

```CMake
if(NOT WIN32)
  option(BUILD_SERVER "Build Snapserver" ON) # no Windows server for now
endif()
```

Pull request are welcome :)

Should be possible to run it in WSL and capture audio from a TCP socket.
